app/models/user.rb

```
@@ -42,6 +42,10 @@ def friends?
   end
 
   def steam_friends
-    Steam::User.friends(uid)
+    begin
+      Steam::User.friends(uid)
+    rescue Steam::JSONError
+      return []
```

We could let this bubble up to the controller and catch it there, or we can simply be like:

> Looks like you don't have any friends. Is your profile private?